source/jormungandr/jormungandr/realtime_schedule/siri.py

```diff
@@ -0,0 +1,184 @@
+            "Content-Length": len(encoded_request)
+        }
+
+        logging.getLogger(__name__).debug('siri RT service, post at {}: {}'.format(self.service_url, request))
+        try:
+            return self.breaker.call(requests.post,
+                                     url=self.service_url,
+                                     headers=headers,
+                                     data=encoded_request,
+                                     verify=False,
+                                     timeout=self.timeout)
+        except pybreaker.CircuitBreakerError as e:
+            logging.getLogger(__name__).error('siri RT service dead, using base '
+                                              'schedule (error: {}'.format(e))
+            self.record_external_failure('circuit breaker open')
+        except requests.Timeout as t:
+            logging.getLogger(__name__).error('siri RT service timeout, using base '
+                                              'schedule (error: {}'.format(t))
+            self.record_external_failure('timeout')
+        except Exception as e:
+            logging.getLogger(__name__).exception('siri RT error, using base schedule')
+            self.record_external_failure(str(e))
+        return None
+
+
+    def _make_request(self, dt, count, message_identifier='StopMonitoringClient:Test:0',
+                      monitoring_ref='Orleans:StopPoint:BP:52600:LOC',
+                      stop_visit_types='all'):
+
+        request = """<?xml version="1.0" encoding="UTF-8"?>
+        <x:Envelope xmlns:x="http://schemas.xmlsoap.org/soap/envelope/"
+                    xmlns:wsd="http://wsdl.siri.org.uk" xmlns:siri="http://www.siri.org.uk/siri">
+          <x:Header/>
+          <x:Body>
+            <GetStopMonitoring xmlns="http://wsdl.siri.org.uk" xmlns:siri="http://www.siri.org.uk/siri">
+              <ServiceRequestInfo xmlns="">
+                <siri:RequestTimestamp>{dt}</siri:RequestTimestamp>
+                <siri:RequestorRef>{RequestorRef}</siri:RequestorRef>
+                <siri:MessageIdentifier>{MessageIdentifier}</siri:MessageIdentifier>
+              </ServiceRequestInfo>
+              <Request version="1.3" xmlns="">
+                <siri:RequestTimestamp>{dt}</siri:RequestTimestamp>
+                <siri:MessageIdentifier>{MessageIdentifier}</siri:MessageIdentifier>
+                <siri:MonitoringRef>{MonitoringRef}</siri:MonitoringRef>
+                <siri:StopVisitTypes>{StopVisitTypes}</siri:StopVisitTypes>
+                <siri:MaximumStopVisits>{count}</siri:MaximumStopVisits>
+              </Request>
+              <RequestExtension xmlns=""/>
+            </GetStopMonitoring>
+          </x:Body>
+        </x:Envelope>
+        """.format(dt=datetime.utcfromtimestamp(dt).isoformat(),
+                   count=count,
+                   RequestorRef=self.requestor_ref,
+                   MessageIdentifier=message_identifier,
```

Comment:
little comments on the meaning of those params would be nice (something from Siri doc?)